gc/base/standard/Scavenger.hpp

```diff
@@ -70,6 +70,11 @@ class MM_Scavenger : public MM_Collector
 	/*
 	 * Data members
 	 */
+public:
+	/* The following start/end times record total cycle duration, done only by main thread. */
+	uint64_t _collectionStartTime;
```

Comment:
Let's name this to `_cycleStart/EndTime` - apologies for switching it on you after suggesting `collection..`. IMO `_cycleStart/EndTime` is less ambiguous, `_collectionStartTime` can infer the `scavengeStart/EndTime` that we are recording for `reportScavengeEnd`.